modules/multicast/multicast.c

```diff
@@ -6,7 +6,9 @@
  * Copyright (C) 2021 Commend.com - c.huber@commend.com
  */
 
+#ifdef HAVE_GETIFADDRS
 #include <net/if.h>
+#endif
```

Comment:
@cHuberCoffee  Not sure. Can't we simply drop this?

Can't find this include in baresip

Reply:
It is needed for IPv6 multicast.